key/src/lib.rs

```diff
@@ -31,6 +31,7 @@ extern crate serde;
 #[macro_use]
 extern crate serde_derive;
 extern crate serde_json;
+extern crate sodiumoxide;
```

Comment:
You don't need to add extern crate statements anymore since 2018 edition.

Reply:
Thank you, I studied the comment on it after you pointed out.